plugins/simple-pivot/src/js/vite.config.js

```diff
@@ -0,0 +1,32 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react-swc';
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => ({
+  build: {
+    minify: false,
+    lib: {
+      entry: './src/index.ts',
+      fileName: () => 'index.js',
+      formats: ['cjs'],
+    },
+    rollupOptions: {
+      external: [
+        'react',
+        'react-dom',
+        'redux',
+        'react-redux',
+        // Externalize all Deephaven dependencies to reduce bundle size and maintain proper context for themes, etc.
+        '@deephaven/components',
+        '@deephaven/iris-grid',
+        '@deephaven/icons',
+        '@deephaven/jsapi-bootstrap',
+        '@deephaven/log',
+        '@deephaven/plugin',
```

Comment:
Put it in alphabetical order, and you can add jsapi-utils to the list of externalized packages
```suggestion
        '@deephaven/components',
        '@deephaven/icons',
        '@deephaven/iris-grid',
        '@deephaven/jsapi-bootstrap',
        '@deephaven/jsapi-utils',
        '@deephaven/log',
        '@deephaven/plugin',
```